pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/ArrayInitializationVerbosenessRule.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.codestyle;
+
+import net.sourceforge.pmd.lang.java.ast.ASTLocalVariableDeclaration;
+import net.sourceforge.pmd.lang.java.rule.AbstractJavaRule;
+
+public class ArrayInitializationVerbosenessRule extends AbstractJavaRule {
+
+    @Override
+    public Object visit(ASTLocalVariableDeclaration node, Object data) {
```

Comment:
be ware, this node may contain more than one variable:

```java
int ar[] = new int[] { 1,2,3}, foo[] = new int[] { 4, 5, 6 }
```

You want to produce 2 violations, one for each initializer. Consider adding a test-case for this scenario.